### PR TITLE
[MM-25652] Add /integrity endpoint to local mode API

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR makes the necessary changes to port the `integrity` command to `mmctl`.
I've also included a long overdue fix to the command itself.

#### Ticket

https://mattermost.atlassian.net/browse/MM-25652

#### Related PRs

https://github.com/mattermost/mattermost-api-reference/pull/550

### CODE REVIEW

File: cmd/mattermost/commands/integrity.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func init() {
 	RootCmd.AddCommand(IntegrityCmd)
 }
 
-func printRelationalIntegrityCheckResult(data store.RelationalIntegrityCheckData, verbose bool) {
+func printRelationalIntegrityCheckResult(data model.RelationalIntegrityCheckData, verbose bool) {
```
Comment (isacikgoz) [inline cmd/mattermost/commands/integrity.go:27]: 💯

File: api4/integrity_local.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (mgdelacroix) [inline api4/integrity_local.go:1]: I think we should move the handler to `system_local.go` so it's placed in the same file as the endpoint itself

Comment (streamer45) [reply api4/integrity_local.go:1]: Absolutely, I think I did back and forth and was not making up my mind. 
Thanks for pointing it out, will change :+1:

**Pull request merged**